stack-package/.registry/resources/samples.upbound.io/mytype/v1alpha1/ui-schema.yaml

```diff
@@ -0,0 +1,4 @@
+uiSpecVersion: 0.3
```

Comment:
which UI schema file wins in this case since there is also a matching `mytype.ui-schema.yaml` file?